docs/_templates/layout.html

```diff
@@ -0,0 +1,322 @@
+{# TEMPLATE VAR SETTINGS #}
+{%- set url_root = pathto('', 1) %}
+{%- if url_root == '#' %}{% set url_root = '' %}{% endif %}
+{%- if not embedded and docstitle %}
+  {%- set titlesuffix = " &mdash; "|safe + docstitle|e %}
+{%- else %}
+  {%- set titlesuffix = "" %}
+{%- endif %}
+{%- set lang_attr = 'en' if language == None else (language | replace('_', '-')) %}
+{% import 'theme_variables.jinja' as theme_variables %}
```

Comment:
Assuming you did not write it yourself: where is it copied from? Does it need to be kept in sync with other places in qiskit?